runtime/development/src/lib.rs

```diff
@@ -1295,6 +1295,46 @@ impl pallet_xcm_transactor::Config for Runtime {
 parameter_types! {
 	pub const MaxActiveLoansPerPool: u32 = 50;
 	pub const MaxWriteOffPolicySize: u32 = 10;
+	pub const MaxHasDispatchedSize: u32 = 1;
+	pub const MaxPools: u32 = 50;
+	pub const MaxPriceOracleMembers: u32 = 10;
+	pub RootOperatorOraclePrice: AccountId = PalletId(*b"orac_pri").into_account_truncating();
```

Comment:
I guess the pallet ID here should be part of https://github.com/centrifuge/centrifuge-chain/blob/main/libs/types/src/ids.rs?

Reply:
Added there, and I've removed the `LOANS_PALLET_ID` because it's no longer used.

This was another pending talk. To be honest, not sure if we want a pallet id as a root account. This account should be handled by us or by root/conceal. I think adding a simple public key account here we do not let us handle this later.